README.md

```diff
@@ -205,9 +210,75 @@ otherwise apply middleware the store by namespace.
 * Return value
   - void
 
-## Advanced use
+## Advanced Use
+
+### Chain react of State
+
+The chain react is a common requirement, you can implement it by calling other Store in Store.
```

Comment:
Store interaction is  a common usage scene which can be implemented by calling actions from other  store in a store's action.

Reply:
done